Sources/GEOSwift/GEOS/GeometryConvertible+GEOS.swift

```diff
@@ -425,6 +425,22 @@ public extension GeometryConvertible {
         }
     }
 
+    func bufferWithStyle(width: Double, quadsegs: Int32 = 8, endCapStyle: Int32 = 1, joinStyle: Int32 = 1, mitreLimit: Double = 5.0) throws -> Geometry? {
```

Comment:
Could you define enums to represent the possible values for endCapStyle and joinStyle? GEOS has similar enums, and we can do something similar in Swift for better safety and call-site readability.

Here's a link to the documentation comment for GEOSBufferWithStyle which mentions the two GEOS enums: https://github.com/GEOSwift/geos/blob/f510e634c822116fca615064d889300dba40d761/Sources/geos/public/geos_c.h#L3398-L3400

Reply:
I can but we don't have any enums like this in GEOSwift. Can you suggest where to put them?

Reply:
Let's just add them at the bottom of this file for now. There's one other one there called `MakeValidMethod` that does something similar.